test/instrumentation/run.sh

```diff
@@ -0,0 +1,11 @@
+# /bin/sh
```

Comment:
fwiw `bash` is generally more widely used. Not a big deal, but you'll likely come across more docs pointing to `bash` code that may not work with `sh`.